meshroom/ui/qml/Viewer/Viewer2D.qml

```diff
@@ -129,14 +129,15 @@ FocusScope {
             }
         }
         onWheel: {
-            var zoomFactor = wheel.angleDelta.y > 0 ? factor : 1/factor
+            var zoomFactor = wheel.angleDelta.y > 0 ? factor : 1/factor;
 
             if(Math.min(imgContainer.width, imgContainer.image.height) * imgContainer.scale * zoomFactor < 10)
-                return
-            var point = mapToItem(imgContainer, wheel.x, wheel.y)
-            imgContainer.x += (1-zoomFactor) * point.x * imgContainer.scale
-            imgContainer.y += (1-zoomFactor) * point.y * imgContainer.scale
-            imgContainer.scale *= zoomFactor
+                return;
+            var point = mapToItem(imgContainer, wheel.x, wheel.y);
+            imgContainer.x += (1-zoomFactor) * point.x * imgContainer.scale;
+            imgContainer.y += (1-zoomFactor) * point.y * imgContainer.scale;
+            imgContainer.scale *= zoomFactor;
+            floatImageViewerLoader.targetSize *= zoomFactor;
```

Comment:
Could we set the scale to the floatImageViewerLoader? To avoid modifying 2 vars that needs to stay in sync.

Reply:
floatImageViewerLoader could have a local var "targetSize: scale * maxImageSize"